test/text_query_test.exs

```diff
@@ -83,4 +101,13 @@ defmodule TextQueryTest do
     {:ok, %{rows: rows}} = Mariaex.query(pid, "SELECT dt FROM test_text_query_table", [], query_type: :text)
     assert(rows == [[{{1,1,1}, {0,0,0,0}}], [{{1,1,1}, {0,0,1,0}}]])
   end
+
+  if System.get_env("JSON_SUPPORT") === "true" do
```

Comment:
Please use `@tag` to skip tests so we can see when tests are skipped, and it allows more flexibility when testing outside CI. This can be done by adding `@tag :json` to the test and `exclude: [json: System.get_env("JSON_SUPPORT") != "true"]` to the `ExUnit.start/1` call in `test/test_helper.exs`.

Any setup can be done in the test as there is only a single test.

Reply:
I fixed via cb8a783 .